packages/wallet/src/redux/reducers/__tests__/closing.test.ts

```diff
@@ -104,7 +105,7 @@ describe('start in WaitForOpponentConclude', () => {
     const fromHexMock = jest.fn().mockReturnValue(concludeCommitment2);
     Object.defineProperty(fmgCore, "fromHex", { value: fromHexMock });
 
-    const action = actions.messageReceived('0x0', '0x0');
+    const action = actions.messageReceived('commitment' as unknown as Commitment, '0x0');
```

Comment:
This will probably need to be changed to `concludeCommitment2` instead of `'commitment' as unknown as Commitment` to get the test passing (also the `fromHex` mocking can probably go)